roles/edpm_nova/tasks/adoption.yml

```diff
@@ -9,10 +9,14 @@
     - name: Check if nova_virtlogd container exists
       ansible.builtin.command: podman ps -a --filter name=nova_virtlogd --format \{\{.Names\}\}
       register: nova_virtlogd_status
+      failed_when: false
       changed_when: false
+      ignore_errors: true
     - name: Stop nova_virtlogd containers not managed by service units
-      ansible.builtin.shell: "podman stop nova_virtlogd '||' true"
+      ansible.builtin.shell: "podman stop nova_virtlogd"
```

Comment:
you can use `ansible.builtin.command` instead of `ansible.builtin.shell` since you don't need any fancy shell feature.

Reply:
done, thanks!